src/openrct2-ui/audio/MemoryAudioSource.cpp

```diff
@@ -159,44 +171,62 @@ namespace OpenRCT2 { namespace Audio
             return result;
         }
 
-        bool Convert(const AudioFormat * format)
+        virtual bool IsLoading() const override
+        {
+            return _isLoading;
+        }
+
+        virtual bool Convert(const AudioFormat * format) override
         {
             if (*format != _format)
             {
-                SDL_AudioCVT cvt;
-                if (SDL_BuildAudioCVT(&cvt, _format.format, _format.channels, _format.freq, format->format, format->channels, format->freq) >= 0)
+                if (true)
```

Comment:
what's the point of this `if`?

Reply:
Disables or enables the threading for now, question will be do we want this. Thats why I left it there.